examples/language/bert/finetune.py

```diff
@@ -165,12 +237,17 @@ def main():
     # bert pretrained model
 
     cfg = AutoConfig.from_pretrained(model_name, num_labels=data_builder.num_labels)
-    if model_name == "bert-base-uncased":
-        model = BertForSequenceClassification.from_pretrained(model_name, config=cfg)
-    elif model_name == "albert-xxlarge-v2":
-        model = AlbertForSequenceClassification.from_pretrained(model_name, config=cfg)
-    else:
-        raise RuntimeError
+
+    # lazy_init
+    use_lazy_init = args.use_lazy_init
+    ctx = LazyInitContext() if use_lazy_init else nullcontext()
```

Comment:
Lazy init context does not support `from_pretrained()` now. It's not essential in this example. You can remove it.